syntax/javascript.vim

```diff
@@ -162,7 +162,7 @@ syntax region  jsBlock                        matchgroup=jsBraces              s
 syntax region  jsModuleGroup        contained matchgroup=jsModuleBraces        start=/{/ end=/}/   contains=jsModuleKeyword,jsModuleComma,jsModuleAs,jsComment skipwhite skipempty nextgroup=jsFrom
 syntax region  jsSpreadExpression   contained matchgroup=jsSpreadOperator      start=/\.\.\./ end=/[,}\]]\@=/ contains=@jsExpression
 syntax region  jsRestExpression     contained matchgroup=jsRestOperator        start=/\.\.\./ end=/[,)]\@=/
-syntax region  jsTernaryIf                    matchgroup=jsTernaryIfOperator   start=/?/  end=/\%(:\|[\}]\@=\)/  contains=@jsExpression extend skipwhite skipempty nextgroup=@jsExpression
+syntax region  jsTernaryIf                    matchgroup=jsTernaryIfOperator   start=/?\_s/  end=/\%(:\|[\}]\@=\)/  contains=@jsExpression extend skipwhite skipempty nextgroup=@jsExpression
```

Comment:
Unfortunately, this suggestion does not work. It breaks the ternary operator's syntax highlighting. Sadly, since this plugin uses regex to parse language keywords, our expressiveness is limited.

With your suggestion:

![image](https://user-images.githubusercontent.com/3723671/27337077-671f1c66-559f-11e7-9ac1-a61269434b0b.png)


With my suggestion:

![image](https://user-images.githubusercontent.com/3723671/27337123-88a30596-559f-11e7-8874-536a3455ad0d.png)